samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/listener/evaluation/HistogramListener.java

```diff
@@ -476,11 +476,13 @@ public boolean histogramScores(HistogramScoresBean histogramScores, TotalScoresB
 					  //totalpossible = totalpossible + item.getScore().doubleValue();
 					  //ArrayList responses = null;
 
+                      // Should be set before determineResults
```

Comment:
odd identation :(

Reply:
I'm a bit conflicted between not confusing github with the indentation and mixing tabs with spaces on the same line
![image](https://github.com/sakaiproject/sakai/assets/53173679/f01749cb-4a0b-4f1f-8cc1-4e6271eb66a8)

Reply:
maybe 5x8+2 spaces? but then it looks bad in the editor...

Reply:
Stick to the same indentation that the file uses :(